HHS/Head-Start-TTADP
# [TTAHUB-533] Cleanup delivery data

## Description of change

Created a script to alter the data some reports have in the delivery type column in the database, based on mappings in the jira.

## How to test
Imports from staging should have the offending reports. Locate them in your DB, run the script, and then confirm that those reports have been edited appropriately, and that the entire database wasn't mucked about with.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-533

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
